sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SparkMetadataOperationSuite.scala

```diff
@@ -396,4 +400,187 @@ class SparkMetadataOperationSuite extends HiveThriftJdbcTest {
       }
     }
   }
+
+  test("Hive ThriftServer JDBC Database MetaData API Auditing") {
```

Comment:
How about splitting this test into the two parts?
```
test("Hive ThriftServer JDBC Database MetaData API Auditing - supported") {
test("Hive ThriftServer JDBC Database MetaData API Auditing - not supported") {
```

Reply:
SGTM